### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by nahnahdev**
## Project Requirements 

- [ ] Set up a new project with webpack that is based on the webpack exercise you have already completed.

- [ ]  Create an `index.htm`l file and write your HTML markup here. Create an empty To Do List placeholder `(<div> or <ul> element)`. The` index.html` file must be set as a template using the HTML Webpack Plugin.
- [ ] Create an `index.js `file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:

1.         description [string].
2.          completed [bool].
3.            index: [number].

- [ ] Write a function to iterate over the tasks array and populate an HTML list item element for each task.

- [ ] On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.

- [ ] Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.
- [ ] All your source files `(index.html, index.js and style.css)` must be located in /src directory and your distribution files will be generated by webpack and served by webpack dev server from /dist folder.

### CODE REVIEW

File: modules/objects.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    string: 'Arange a call',
+    boo1: 'Join stand-up Call',
+    id: '3',
+  },
+  {
+    string: 'Control the teams project',
+    boo1: 'Join stand-up Call',
+    id: '4',
+  },
+];
```
Comment (roshan-bajgain) [inline modules/objects.js]: - [ ] According to the requirement, you need to set an array of some simple to-do tasks (array of objects). Each task object should contain three keys. that is: `description [`string`], completed [`bool`], index: [`number`]`. Please make it like this.
`const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
`

File: modules/populateFunction.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+const populate = () => {
+  tasks.forEach((element) => {
+    const add = document.createElement('div');
+    add.dataset.elementid = element.id;
+    add.innerHTML = `
+  <div class="container">
+  <span><input type='checkbox'>${element.string}</span>
+  <div><i class="fa-solid fa-ellipsis-vertical"></i></div>
+  </div><hr>
```
Comment (roshan-bajgain) [inline modules/populateFunction.js]: - [ ] I would suggest you please make sure to populate the HTML list item element for each task. You have populated the string but make sure to populate the `completed [bool]`. For reference:
``` <p>${completed ? 'completed' : 'not completed'}</p>`; ```

File: README.md [hunk 3]
```diff
@@ -1 +1,68 @@
...
+
+### Setup
+
+- If it suits you, on your local computer, create a directory in your preferred location (e.g in the Desktop or Document directory)
+
+### Install
+
+- Open your Terminal/Powershell or any command line interface you use, then navigate to your created above directory or the directory you'll like to clone this project directory.
+- Type `https://github.com/NahnahAJ/To-Do-List.git` if your're using https with git or `git@github.com:NahnahAJ/To-Do-List.git` if your preferred is ssh and hit enter.
+- Navigate to the `To-Do List` directory on your local computer in your above chosen directory and open the `src/index.html` file.
```
Comment (roshan-bajgain) [inline README.md]: - [ ] I would suggest you please make sure that you write the proper step to run the project on the machine.
 For example, You need to do `npm install` then only you have to do `npm start` to start the project. please write those step

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @NahnahAJ, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
Status: Approved ✔️
Hi @NahnahAJ 

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations! 🎉💯🌟

Good Points 👍
- Descriptive Pull Request
- No linter errors

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**